### DISCUSSION TITLE
Live plot crashes with additional setpoints

### DISCUSSION THREAD

**Discussion by peitian1020** [microsoft/Qcodes#3767] [Q&A]
I am using doNd to do measurements with several standalone parameters. To record these parameters for future analysis, I put additonal_setpoints in doNd, but it seems to be not compatible with live plot. The plot failed sometimes, and it turned out the more parameters there are, the more likely it crashes. It is only live plot not working, measurement can still be done and all parameters can be written properly to the dataset.

**Comment by jenshnielsen** [comment]
Thanks for the question @peitian1020 I am a little unsure which plot you mean. Do you mean the plot that dond generates it self (which is not live) or a plot from plottr ?

**Reply by peitian1020** [reply]
I mean the live plot. the plot generated by dond can still be created when the measurements are done, and if I do not use live plot, no errors will happen.

**Reply by jenshnielsen** [reply]
Ok in that case this seems like an issue that should be raised in the plottr issue tracker since it sounds like the issue is that plottr does not know how to handle plots with extra dimensions

**Reply by peitian1020** [reply]
Exactly, the more additional setpoint parameters I use, so the larger the dimension is, the more likely errors happen.
